hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/ScmConfigKeys.java

```diff
@@ -561,6 +561,10 @@ public final class ScmConfigKeys {
   public static final long OZONE_SCM_HA_RATIS_SNAPSHOT_THRESHOLD_DEFAULT =
           1000L;
 
+  public static final String OZONE_SCM_HA_RATIS_SNAPSHOT_GAP
+      = "raft.server.snapshot.creation.gap";
```

Comment:
Could you refine the name and add some comments to help with the comprehension for the other reader? It might be confusing with the AutoTriggerThreshold as it is only used when RATIS ```takeSnapshotAsync``` by manual. How about "raft.server.snapshot.manual.creation.gap"?